next.config.js

```diff
@@ -1,6 +1,6 @@
 const nextConfig = {
   images: {
-    domains: ['134.209.25.111', '172.19.0.7'],
+    domains: ['134.209.25.111', '172.19.0.7', '172.19.0.2'],
```

Comment:
`172.17. 0.0/16` is the Docker's default subnet, `'172.19.0.7', '172.19.0.2'` should be removed from this list.